src/libraries/Microsoft.PowerFx.Core/Public/Values/RecordValue.cs

```diff
@@ -109,6 +109,16 @@ public async IAsyncEnumerable<NamedValue> GetFieldsAsync([EnumeratorCancellation
                 Func<Task<FormulaValue>> getFormulaValue = async () => await GetFieldAsync(fieldName, cancellationToken).ConfigureAwait(false);
                 yield return new NamedValue(fieldName, getFormulaValue, backingDType);
             }
+        }
+
+        /// <summary>
+        /// Returns only the actual fields present in the record. This prevents fields from wrappers.

```

Comment:
>This prevents fields from wrappers [](http://example.com/codeflow?start=66&length=34)

It's a little confusing. Does it mean "This prevents fields that are not defined in the record, but have *implicit* blank values because of the type, from being returned."?

Reply:
Thats correct. I'll change the description since your suggestion is better.